docs/sources/operations/storage/wal.md

```diff
@@ -62,7 +62,7 @@ You should not target 100% disk utilisation.
 
 ## Migrating from stateless deployments
 
-The ingester _deployment without WAL_ and _statefulset with WAL_ should be scaled down and up respectively in sync without transfer of data between them to ensure that any ingestion after migration is reliable immediately.
+The ingester _deployment without WAL_ and _StatefulSet with WAL_ should be scaled down and up respectively in sync without transfer of data between them to ensure that any ingestion after migration is reliable immediately.
```

Comment:
Deployment should also be capitalized but we don't have it the Vale rule given the likelihood of false positives.

https://github.com/grafana/writers-toolkit/blob/main/vale/Grafana/Kubernetes.yml#L13-L15

```suggestion
The ingester _Deployment without WAL_ and _StatefulSet with WAL_ should be scaled down and up respectively in sync without transfer of data between them to ensure that any ingestion after migration is reliable immediately.
```